openstates/in/bills.py

```diff
@@ -45,198 +45,185 @@ class INBillScraper(BillScraper):
+        all_pages = client.unpaginate(r)
+        for b in all_pages:
+            bill_id = b["billName"]
+            bill_link = b["link"]
+            api_source = api_base_url + bill_link
+            bill_json = client.get("bill",session=session,bill_id=bill_id.lower())
+            
+            title = bill_json["title"]
+            #sometimes title is blank
+            #if that's the case, we can check to see if
+            #the latest version has a short description
+            if not title:
+                title = bill_json["latestVersion"]["shortDescription"]
+
+            #and if that doesn't work, use the bill_id but throw a warning
+            if not title:
+                title = bill_id
+                self.logger.warning("Bill is missing a title, using bill id instead.")
+
+
+            original_chamber = "lower" if bill_json["originChamber"].lower() == "house" else "upper"
+            bill = Bill(session,original_chamber,bill_id,title)
+            
+            bill.add_source(api_source,note="API key needed")
+            bill.add_source(self.make_html_source(session,bill_id))
+            #documents/versions
+            #todo - these are terrible.
+
+
+            #sponsors
+            positions = {"Representative":"lower","Senator":"upper"}
+            for s in bill_json["authors"]:
+                bill.add_sponsor("primary",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="author")
+
+            for s in bill_json["coauthors"]:
+                bill.add_sponsor("cosponsor",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="coauthor")
+
+            for s in bill_json["sponsors"]:
+                bill.add_sponsor("primary",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="sponsor")
+
+            for s in bill_json["cosponsors"]:
+                bill.add_sponsor("cosponsor",
+                    self.get_name(s),
+                    chamber=positions[s["position_title"]],
+                    official_type="cosponsor")
+
+            #actions
+            action_link = bill_json["actions"]["link"]
+            api_source = api_base_url + action_link
+            actions = client.get("bill_actions",session=session,bill_id=bill_id.lower())
+            for a in actions["items"]:
+                action_desc = a["description"]
+                chamber = "lower" if a["chamber"]["name"].lower() == "house" else "upper"
+                date = a["date"]
+                date = datetime.datetime.strptime(date,"%Y-%m-%dT%H:%M:%S")
+                
+                action_type = []
+                d = action_desc.lower()
+                committee = None
+                if "first reading" in d:
+                    action_type.append("bill:reading:1")
```

Comment:
added this and a check to make sure we're referring to a bill reading, not an amendment.